fontbe/src/post.rs

```diff
@@ -0,0 +1,35 @@
+//! Generates a [post](https://learn.microsoft.com/en-us/typography/opentype/spec/post) table.
+
+use fontdrasil::orchestration::Work;
+use read_fonts::types::{FWord, Fixed};
+use write_fonts::tables::post::Post;
+
+use crate::{
+    error::Error,
+    orchestration::{BeWork, Context},
+};
+
+struct PostWork {}
+
+pub fn create_post_work() -> Box<BeWork> {
+    Box::new(PostWork {})
+}
+
+impl Work<Context, Error> for PostWork {
+    /// Generate [post](https://learn.microsoft.com/en-us/typography/opentype/spec/post)
+    fn exec(&self, context: &Context) -> Result<(), Error> {
+        // TODO a more serious post
+        let post = Post::new(
```

Comment:
where is the formatType = 3.0 defined in here? Or does write-fonts take care of setting that automatically?

Reply:
hm I inspected the `post.table` file you compiled, and the format is 1.0, not 3.0: the first four bytes are `b'\x00\x01\x00\x00'`

Reply:
post 1.0 is for fonts that contains "exactly the 258 glyphs in the standard Macintosh TrueType font file", which is not the case for you. You want formatType=3.0

Reply:
Erk good catch. For some reason I thought that would get me 3 :(